README.md

```diff
@@ -3,38 +3,45 @@
 [![Built with Foundry](https://img.shields.io/badge/Built%20with-Foundry-FF6E3D?logo=ethereum)](https://book.getfoundry.sh/)
 
 **An OSx governance plugin, enabling immediate voting through token locking**
+
 Built on Aragon OSx's modular framework, LockToVote redefines DAO participation by eliminating the need for *ahead of time* token snapshots with an [IVotes](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/governance/utils/IVotes.sol) compatible token. Any vanilla ERC20 can now be used to participate in DAO governance.
 
-## Two flavours
+See the [ERC20 token checklist](#erc20-token-checklist) below.
+
+## Overview
 
 ![Overview](./img/overview.png)
 
-### `LockToVote`
+`LockToVote` is a versatile majority voting plugin with configurable modes:
 
-Versatile majority voting plugin with configurable modes:
 - **Multi-option voting**: Vote Yes/No/Abstain
 - **Three voting modes**:
-  - **Vote Replacement**: Update your vote option mid-proposal, unlock early
-  - **Early Execution**: Automatically execute proposals when thresholds are mathematically irreversible
+  - **Vote Replacement**: Update your vote option mid-proposal
   - **Standard Mode**: Traditional voting with append-only allocations
 - **Customizable thresholds**: Minimum participation, support threshold, and a certain approval tally
 
 ## Architecture Overview
 
 ### Core Components
 
-1. **LockManager**: The custodial contract managing token locks and allowing to vote in multiple proposals with a single lock
-   - `LockManagerBase` contains the common logic, while `LockManagerERC20` includes the specific implementation to manage ERC20 token locks.
-   - `lock()` deposits the current ERC20 allowance into the contract and updates the cumulative locked balance
-   - `vote()` allow users to use the currently locked balance on a given proposal
-   - Locking and voting can be done at once with `lockAndVote()`
-   - To prevent unlocking with votes on active proposals, it keeps track of them via the `proposalCreated()` and `proposalEnded()` hooks
+#### LockManager
+
+The custodial contract managing token locks and allowing to vote in multiple proposals with a single lock
```

Comment:
@brickpop probably it's worth also mentioning that now if you create a proposal, the locked amount of token will stay locked until the proposal voting period has ended.